src/main/java/io/nats/client/Nats.java

```diff
@@ -131,6 +131,28 @@ public static Connection connect(String url) throws IOException, InterruptedExce
         return createConnection(options, false);
     }
 
+    /**
+     * Connect to the specified URL with the specified username, password and default options.
+     *
+     * <p>This is a synchronous call, and the connection should be ready for use on return
+     * there are network timing issues that could result in a successful connect call but
+     * the connection is invalid soon after return, where soon is in the network/thread world.
+     *
+     * <p>If the connection fails, an IOException is thrown
+     *
+     * <p>See {@link Nats#connect(Options) connect(Options)} for more information on exceptions.
+     *
+     * @param url the url of the server, ie. nats://localhost:4222
```

Comment:
To avoid this not being used when you want multiple urls we should mention that url can be a comma separated list of servers as well.
nats://localhost:4222,nats://localhost:4223
createConnection splits around ,

Reply:
updated javadoc